linera-rpc/src/grpc/server.rs

```diff
@@ -508,7 +508,7 @@ where
         }
     }
 
-    #[instrument(target = "grpc_server", skip_all, err, fields(nickname = self.state.nickname(), chain_id = ?request.get_ref().chain_id(), ?request))]
+    #[instrument(target = "grpc_server", skip_all, err, fields(nickname = self.state.nickname(), chain_id = ?request.get_ref().chain_id()))]
     async fn handle_certificate(
         &self,
```

Comment:
@andresilva91 Can we add `tracing::trace!(?certificate, "Handling certificate");` ?

Reply:
@Twey's PR #2295 did not create the issue but it explicitly maintained the bug :)